arabic/dream/dream.md

```diff
@@ -2,5 +2,5 @@
 احلم......الاحلام تتحقق اذا صدقة النية
 ولكن الاحلام لا تتحقق الا بالتوكل على الله والاخذ بالاسباب
 الان انت فى بداية الطريق ماذا ستفعل
-1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md
+1- [ستبدأ فى التعلم] (https://github.com/udacity/create-your-own-adventure/blob/master/arabic/learn/learn.md)
 1-[ستبدأ فى التعلم](../learn/learn.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.